rust_src/build.rs

```diff
@@ -127,9 +127,15 @@ impl<'a> ModuleParser<'a> {
     }
 
     pub fn run(&mut self, in_file: impl BufRead) -> Result<ModuleData, BuildError> {
```

Comment:
Can this be extended to include clippy warnings as well? Because I do not think they work currently.

Reply:
I don’t quite understand what you mean. Can you elaborate a bit?